e2ev2/src/circle-run-tests.sh

```diff
@@ -21,6 +21,9 @@ BKT_ROOT=gs://all-of-us-workbench-test.appspot.com/circle-failed-tests
 FAILED_TESTS=$(gsutil cat $BKT_ROOT/\*.$CIRCLE_SHA1.txt || echo -n)
 gsutil rm $BKT_ROOT/\*.$CIRCLE_SHA1.txt || true
 
+# This warmup can take over thirty seconds, which causes timeouts in tests.
+time (curl https://pr-$PR_SITE_NUM-dot-api-dot-all-of-us-workbench-test.appspot.com; echo)
```

Comment:
I believe Codacy is complaining about PR_SITE_NUM here, which is just not going to have spaces and it doesn't make sense to plan for that case.